packages/wallet/src/redux/types.ts

```diff
@@ -0,0 +1,4 @@
+export const enum WalletProcedure {
```

Comment:
This might not be the best place for this enum but I think it's better than being in the actions file. We can move as needed.